test_scripts/API/Navigation/OnWayPointChange/017_OnWayPointChange_SeveralNotificationsWithIdenticalParams_AllSuccess.lua

```diff
@@ -0,0 +1,78 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_requirements/issues/28
+-- Use case: https://github.com/smartdevicelink/sdl_requirements/blob/master/detailed_docs/embedded_navi/Notification_about_changes_to_Destination_or_Waypoints.md
+-- Item: Use Case 1: Main Flow
+--
+-- Requirement summary:
+-- [OnWayPointChange] As a mobile application I want to be able to be notified on changes
+-- to Destination or Waypoints based on my subscription
+--
+-- Description:
+-- In case:
+-- 1) SDL and HMI are started, Navi interface and embedded navigation source are available on HMI,
+--    mobile applications are registered on SDL and subscribed on destination and waypoints changes notification
+-- 2) Any change in destination or waypoints is registered on HMI (user set new route, canselled the route,
+--    arrived at destination point or crossed a waypoint) and HMI sends several fake params
+
+-- SDL must:
+-- 1) Transfer the notification about changes to destination or waypoints to mobile application without fake params
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/Navigation/commonNavigation')
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
```

Comment:
@GetmanetsIrina Module 'commonFunctions' is unused

Reply:
Removed